themes-default/slim/views/partials/showheader.mako

```diff
@@ -94,45 +75,45 @@
                 </div>
                 <div id="show-rating" class="pull-left col-lg-9 col-md-9 col-sm-12 col-xs-12">
-                 % if 'rating' in show.imdb_info:
-                     <% rating_tip = str(show.imdb_info['rating']) + " / 10" + " Stars" + "<br>" + str(show.imdb_info['votes']) + " Votes" %>
-                     <span class="imdbstars" qtip-content="${rating_tip}">${show.imdb_info['rating']}</span>
-                 % endif
-                 % if not show.imdb_id:
-                     <span>(${show.start_year}) - ${show.runtime} minutes - </span>
-                 % else:
-                     % if 'country_codes' in show.imdb_info:
-                         % for country in show.imdb_info['country_codes'].split('|'):
-                                 <img src="images/blank.png" class="country-flag flag-${country}" width="16" height="11" style="margin-left: 3px; vertical-align:middle;" />
-                         % endfor
-                     % endif
-                                 <span>
-                     % if show.imdb_info.get('year'):
-                                     (${show.imdb_info['year']}) -
-                     % endif
-                                     ${show.imdb_info.get('runtimes') or show.runtime} minutes
-                                 </span>
-                                 <app-link href="http://www.imdb.com/title/${show.imdb_id}" title="http://www.imdb.com/title/${show.imdb_id}">
-                                     <img alt="[imdb]" height="16" width="16" src="images/imdb.png" style="margin-top: -1px; vertical-align:middle;"/>
-                                 </app-link>
-                 % endif
-                % if show.externals.get('trakt_id'):
-                    <app-link href="https://trakt.tv/shows/${show.externals.get('trakt_id')}" title="https://trakt.tv/shows/${show.externals.get('trakt_id')}">
+                    <span v-if="show.rating.imdb && show.rating.imdb.rating"
+                        class="imdbstars"
+                        :qtip-content="show.rating.imdb.rating + ' / 10 Stars<br> ' + show.rating.imdb.votes + ' Votes'"
+                    >
+                        <span :style="{ width: (Number(show.rating.imdb.rating) * 12) + '%' }"></span>
+                    </span>
+                    <template v-if="!show.id.imdb">
+                        <span v-if="show.year && show.year.start">({{ show.year.start }}) - {{ show.runtime }} minutes - </span>
+                    </template>
+                    <template v-else>
+                        <template v-if="show.country_codes && show.country_codes.length >= 1 ">
+                            <img v-for="country in show.country_codes" src="images/blank.png" :class="['country-flag', 'flag-' + country]" width="16" height="11" style="margin-left: 3px; vertical-align:middle;" />
+                        </template>
+                        <span>
+                        % if show.imdb_info.get('year'):
```

Comment:
I don't understand why IMDB info overrides the indexer info on the API data...
I made one change (to the show plot) - #4353, but I'm not even sure that was the correct way to fix it.
In this case, [it's merged into `show.year.start`](https://github.com/pymedusa/Medusa/blob/feature/vueify-showheader/medusa/tv/series.py#L1986)

Reply:
I don't know the answer to that. I wasn't here when the decision was made to join them. I don't know why it was done in the first place...

Reply:
Did you want to check git blame? Depending on who added it we may be able to work out the reason.

For example if it was me then who the hell knows why I did that.

Reply:
Okay yeah let's split them, if you're okay with that? I think the current indexer's data should really be used there as we can fall back to either null or IMDB in the frontend. I don't think we should hardcode it in the API.

Reply:
I think we should take this one step further. Lets add a Structure where each attribute is prepended with the indexer id.

That way in future users can choose which indexer they want to use for, for example the genre, summary or poster.

Reply:
But it should be additional if we choose for it, so we dont risk refactoring for now?